src/test/java/LoginPageTest.java

```diff
@@ -1,41 +1,53 @@
+import io.qameta.allure.Description;
+import io.qameta.allure.Feature;
+import io.qameta.allure.Issue;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.chrome.ChromeDriver;
-import org.openqa.selenium.support.ui.ExpectedConditions;
-import org.openqa.selenium.support.ui.WebDriverWait;
 import org.testng.Assert;
-import org.testng.annotations.AfterClass;
-import org.testng.annotations.BeforeClass;
-import org.testng.annotations.Test;
+import org.testng.annotations.*;
 
 import java.util.concurrent.TimeUnit;
 
 import static org.testng.Assert.assertTrue;
 
-public class LoginPageTest {
-    private WebDriver driver = new ChromeDriver();
-    private LoginPage log = new LoginPage(driver);
+
+public class LoginPageTest extends BaseClass {
+    private WebDriver dr = getDriver();

```

Comment:
driver
loginPage
It is important to give correct names, because when I noticed in before hook "log.get()" I thought that you getting logger